dictionary/views.py

```diff
@@ -1,3 +1,27 @@
+'''
+Views for the dictionary application.
+'''
+
 from django.shortcuts import render
+from django.http import HttpResponseRedirect
+from django.contrib.postgres.search import SearchVector
+
+from . import forms
+from . import models
+
+def search(request):
+    '''
+    This is a first crack at a basic search interface.
+    '''
+    if request.method == 'POST':
+        form = forms.SearchForm(request.POST)
+        if form.is_valid():
+            search_text = form.cleaned_data['search_text']
+            search_vector = SearchVector('simple', 'traditional', 'pronunciation')
+            results = models.Entry.objects.annotate(search=search_vector).filter(search=search_text)
+            form = forms.SearchForm()
+            return render(request, 'dictionary/search.html', {'form': form, 'results': results,})
+    else:
+        form = forms.SearchForm()
```

Comment:
This is definitely on the list of things to do as is improving the general search capabilities.  Have you ever used the fuzzy search in PostgreSQL from Django's ORM?  

I tried the trigram search just because it was there, but it does not work very well for this application.  I really need a minimum edit distance or something similar.  

I saw a third party module for this but I don't know if it's well supported.  Alternatively, I could try to build out some custom functionality.